gulpfile.js

```diff
@@ -117,10 +122,17 @@ gulp.task('dev', function (cb) {
 
     runSequence('build-dev', 'server', cb);
 });
+
 gulp.task('prod', function (cb) {
     WATCH = true;
 
     runSequence('build-prod', 'server', cb);
 });
 
+gulp.task('ci', ['build-dev'], function () {
+    return gulp.src(['dist/**', '!dist/client.zip'])
+        .pipe(zip('client.zip'))
+        .pipe(gulp.dest('dist'));
+});
+
```

Comment:
Should we do a clean here? We don't really have a clean as part of the client build but probably not a bad idea at some point. That said, I'm assuming the CI starts with a clean slate each time?